prow/cmd/peribolos/main.go

```diff
@@ -692,7 +692,7 @@ func orgInvitations(opt options, client inviteClient, orgName string) (sets.Stri
 		if i.Login == "" {
 			continue
 		}
-		invitees.Insert(i.Login)
+		invitees.Insert(github.NormLogin(i.Login))
```

Comment:
lol I am the best test case I assume /s 👍

Reply:
Thanks for breaking everything with your silly capitals 🤣